server.js

```diff
@@ -20,4 +23,64 @@ app.get("/", function (request, response) {
   response.sendFile(__dirname + "/index.html");
 });
 
+//Create a new message
+app.post("/messages", (req, res) => {
+  let newPost = {
+    id: messages.length,
+    from: req.body.from,
```

Comment:
consider using object destructuring to make your code a little more readable. For example:

```js
const {from, text} = req.body

  const newPost = {
    id: messages.length,
    from,
    text
  }
```